SQL/Archive/19.0/2017-11-29-Drop_tarchive_find_new_uploads.sql

```diff
@@ -0,0 +1,4 @@
+-- UNCOMMENT THE FOLLOWING STATEMENT IF tarchive_find_new_uploads IS EMPTY
+-- THIS TABLE IS NOT USED ANYMORE AND WILL NEVER BE FILLED
+
+-- DROP TABLE IF EXISTS tarchive_find_new_uploads;
```

Comment:
I don't think this should be commented out.. there's no real data in the table, and no one uses it. It's better to not have hidden things that people need to know they need to do separately when upgrading unless there's a strong justification for it.